app.js

```diff
@@ -12,14 +12,31 @@ app.set('views', path.join(__dirname, 'views'));
 
 app.use(express.static(path.join(__dirname, 'public')));
 
-// Register the location for handlebars partials here:
-
-// ...
+hbs.registerPartials(path.join(__dirname, 'views/partials'));
 
 // Add the route handlers here:
 
 app.get('/', (req, res) => {
   res.render('index');
 });
 
-app.listen(3000, () => console.log('🏃‍ on port 3000'));
+app.get('/beers', (req, res) => {
+  punkAPI
+    .getBeers()
+    .then(beersFromApi => {
+      res.render('beers', { beersFromApi });
+      console.log('Beers from the database: ', beersFromApi);
+    })
+    .catch(error => console.log(error));
+});
+
+app.get('/random-beers', (req, res) => {
+  punkAPI
+    .getRandom()
+    .then(randomBeer => {
+      res.render('random-beers', { randomBeer });
+    })
+    .catch(error => console.log(error));
+});
+
+app.listen(8000, () => console.log('running‍ on port ${PORT}'));
```

Comment:
Just wanted to mention that you could go with the common practice of storing your port number in a variable `const PORT = 8000;` then refer to it `app.listen(PORT, () => console.log(`Running on port ${PORT}`))`